cmd/config_dirigent_dandelion_rps.json

```diff
@@ -8,6 +8,10 @@
   "DirigentControlPlaneIP": "10.0.1.253:9091",
   "BusyLoopOnSandboxStartup": false,
 
+  "AsyncMode": false,
+  "AsyncResponseURL": "10.0.1.3:8082",
+  "AsyncWaitToCollectMin": 1,
```

Comment:
Done. In one of the next PRs I will extract all Dirigent specific configs into a separate configuration so we don't have a mess, since the number of Dirigent-specific settings has grown a lot.